Viewer/src/net/relinc/viewer/GUI/VideoCorrelationGUI.java

```diff
@@ -83,6 +91,11 @@ public void handle(ActionEvent event) {
 			public void handle(ActionEvent event) {
 				//button shouldn't get clicked unless sample has images.
 				Sample currentSample = getCheckedSamples().get(0);
+				if(currentSample.getResults().size() != 1) {
+					Dialogs.showErrorDialog("Can only have one sample result for this feature", homeController.stage);
```

Comment:
This due to the line tracking video state?

Reply:
yeah. can only be one displacement dataset